### PR TITLE
fix: solve the issue of initial minimum zoom scale is not working

### LINKED ISSUE #189: minimumZoomScale not auto applied automatically?
I tried this out:

```swift
config.cropViewConfig.minimumZoomScale = CGFloat(5)
```

and when selecting a picture, the cropping box still feels like it's at `CGFloat(1)`:

Screen Shot 2022-07-22 at 12 33 16

Should not it auto zoom into the picture along the center?

### PR DESCRIPTION
**Pull request by guoyingtao**
This PR fixes #189

### CODE REVIEW

File: Example/Podfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ target 'MantisExample' do
 
   # Pods for MantisExample
    pod 'Mantis', :path => '../'
+#  pod 'Mantis', '~> 2.0.1'
```
Comment (vanniktech) [inline Example/Podfile:10]: Remove?

File: Sources/Mantis/CropView/CropScrollView.swift [hunk 2]
```diff
@@ -104,10 +107,14 @@ class CropScrollView: UIScrollView {
     
     func resetBy(rect: CGRect) {
         // Reseting zoom need to be before resetting frame and contentsize
-        minimumZoomScale = 1.0
-        zoomScale = 1.0
-
+        minimumZoomScale = max(1.0, initialMinimumZoomScale)
```
Comment (vanniktech) [inline Sources/Mantis/CropView/CropScrollView.swift:110]: Let's document that `minimumZoomScale` needs to be `>= 1.0`?

**Review by vanniktech — APPROVED** [review]

File: Example/Podfile (hunk 1 above)
Comment (guoyingtao) [reply Example/Podfile]: Removed

File: Sources/Mantis/CropView/CropScrollView.swift (hunk 2 above)
Comment (guoyingtao) [reply Sources/Mantis/CropView/CropScrollView.swift:110]: Added a comment for it

**Pull request merged**